Library/Homebrew/system_config.rb

```diff
@@ -156,6 +163,7 @@ def dump_verbose_config(f = $stdout)
 
       f.puts hardware if hardware
       f.puts "Homebrew Ruby: #{describe_homebrew_ruby}"
+      f.puts "System Ruby: #{system_ruby_version}" unless RUBY_PATH == "/usr/bin/ruby"
```

Comment:
```suggestion
      f.puts "System Ruby: #{system_ruby_version}" if RUBY_PATH != SYSTEM_RUBY_PATH
```

and define `SYSTEM_RUBY_PATH` above.

Reply:
> 

> and define `SYSTEM_RUBY_PATH` above

will do